src/commands/executeFunction/eventGrid/EventGridCodeLensProvider.ts

```diff
@@ -12,6 +12,7 @@ export class EventGridCodeLensProvider implements CodeLensProvider {
         firstLineLens.command = {
             title: localize('saveExecute', 'Save and execute'),
             command: 'azureFunctions.eventGrid.sendMockRequest',
+            arguments: ['codeLens']
```

Comment:
It's worth to just make `codeLens` a constant in our `constant.ts` file and importing everywhere it's used.